src/main/TreeQuery.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Blazorise;
+using ei8.Cortex.Graph.Common;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web; // Add this for HttpUtility.UrlDecode
+
+namespace ei8.Cortex.Diary.Plugins.Tree
+{
+    public class TreeQuery
+    {
+        [QueryKey(null)]
+        public IEnumerable<Guid> Postsynaptics { get; set; }
+
+        [QueryKey(null)]
+        public Guid RegionId { get; set; }
```

Comment:
Please change this to Guid? since Guid.Empty is still different from a null Guid.